tools/database/src/run_migrations.rs

```diff
@@ -0,0 +1,27 @@
+use std::path::Path;
+
+/// This can potentially support db specified not in config, but in command line.
+/// `ChangeRelative { path: Path, archive: bool }`
+/// But it is a pain to implement, because of all the current storage possibilities.
+/// So, I'll leave it as a TODO(posvyatokum): implement relative path DbSelector.
```

Comment:
cc @posvyatokum 
Your team mate is sneaking in a TODO for you here ;)

Reply:
That was bad copy-paste. Fixed.